lib/mongoid/persistable/updatable.rb

```diff
@@ -160,6 +165,19 @@ def update_document(options = {})
           end
         end
       end
+
+      # If there is a touch option and it is false, call the timeless method so
+      # that the updated_at attribute is not updated. Call the timeless method
```

Comment:
Call*s*? Unclear whether this says this method will call `timeless` or the caller of this method needs to do that.

Reply:
done